File: atcoder/go/abc230/a.go
package main

import (
	"fmt"
)

func main() {
	var x int
	fmt.Scan(&x)
	if x >= 42 {
		fmt.Printf("AGC%03d\n", x+1)
	} else {
		fmt.Printf("AGC%03d\n", x)
	}
}

File: atcoder/java/abc230/a.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        try(Scanner sc = new Scanner(System.in)) {
            int n = sc.nextInt();
            if(n>=42) {
                n++;
            }
            System.out.println("AGC"+String.format("%03d", n));
        }
    }
}